.storybook/preview.js

```diff
@@ -24,9 +24,9 @@ export const parameters = {
 };
 
 export const decorators = [
-  Story => (
+  story => (
```

Comment:
you know, we can probably get rid of this decorator, there are story options that do this now using layout: center
https://storybook.js.org/docs/react/configure/story-layout

Reply:
Will investigate this option later for the decorator I'll be adding to support scrolling in all stories